wire-ios-request-strategy/Sources/APIs/UserClientAPI.swift

```diff
@@ -45,12 +45,8 @@ class UserClientAPIV0: UserClientAPI {
             apiVersion: apiVersion)
 
         let response = await httpClient.send(request)
-
-        switch response.result {
-        case .success:
-            break
-        default:
-            let _: Payload.UserClient = try mapResponse(response)
+        if response.result != .success {
+            try mapFailureResponse(response)
```

Comment:
something is bothering me with this name, I have the impression we don't use the result, what about:
```suggestion
            let error = mapFailureResponse(response)
            throw error
```

Reply:
or alternatively rename the function to `throwFailureResponse`